pkg/machine/e2e/init_windows_test.go

```diff
@@ -44,8 +44,10 @@ var _ = Describe("podman machine init - windows only", func() {
 		Expect(err).ToNot(HaveOccurred())
 		Expect(inspectSession).To(Exit(0))
 		Expect(inspectSession.outputToString()).To(Equal("true"))
-	})
 
+		// Ensure port 2222 is open
+		Expect(isSSHPortOpen()).To(Equal(true))
```

Comment:
this gives horrible/unusable error messages when it fails. Expect true to be false... which gives a log reader 0 clue what went wrong.
please just inline the function and do a better
```
Expect(err).To(HaveOccurred())
```
match, see the other comment

Reply:
thanks, modified.